README.md

```diff
@@ -57,7 +57,7 @@ Below are the steps needed to start using the System.Web adapters with your ASP.
 
 ## Known Limitations
 
-Below are some of the limitations of the APIs in the adapters. These are usually due to building off of types used in ASP.NET Core that cannot be fully implemented in ASP.NET Core. In the future, analyzers may be used to flag usage to recommend better patterns.
+Below are some of the limitations of the APIs in the adapters. These are usually due to building off of types used in ASP.NET Framework that cannot be fully implemented in ASP.NET Core. In the future, analyzers may be used to flag usage to recommend better patterns.
```

Comment:
```suggestion
Below are some of the limitations of the APIs in the adapters. These are usually due to building off of types used in ASP.NET Core that cannot fully implement the shape of the API in  ASP.NET Framework. In the future, analyzers may be used to flag usage to recommend better patterns.
```

Good catch, although not sure if your edit helps clarify it. How about this?

Reply:
I agree, this way is easier to understand.